data/core.yaml

```
@@ -788,6 +788,10 @@ en:
       inch: in
     max_length_reached: "This string is longer than the maximum length of {maxChars} characters. Anything exceeding that length will be truncated."
     set_today: "Sets the value to today."
+    set_photo_from_viewer: "Store this photo on the currently selected map object"
```

```suggestion
    set_photo_from_viewer: "Tag this photo id on the currently selected map object"
```

"Store" suggest an upload / file. But it is only the id that is referenced as a tag. Maybe like this?